src/subdataframe/subdataframe.jl

```diff
@@ -51,14 +51,14 @@ immutable SubDataFrame{T <: AbstractVector{Int}} <: AbstractDataFrame
     parent::DataFrame
     rows::T # maps from subdf row indexes to parent row indexes
 
-    @compat function SubDataFrame{T}(parent::DataFrame, rows::T) where T
+    function (::Type{SubDataFrame{T}}){T}(parent::DataFrame, rows::T)
```

Comment:
Is this the new recommended syntax for inner constructors? I'm still confused about this, and the manual doesn't help.